txsni/snimap.py

```diff
@@ -52,6 +79,7 @@ class _ConnectionProxy(object):
     def __init__(self, original, factory):
         self._obj = original
         self._factory = factory
+        self._acme_tls_1 = False
```

Comment:
One of the callbacks fails if I only store _acme_tls_1 on the _ConnectionProxy, by adding _acme_tls_1 to the list in __setattr__